src/components/SearchItemList.js

```diff
@@ -1,7 +1,13 @@
 import React from "react";
+import Header from './Header';
 
 const SearchItemList = props => {
-    return <div>Search items page.</div>
+    return (
+        <>
+            <Header />
+            {/*your code goes from here */}
```

Comment:
you can update your component below my comment to see if it works

Reply:
Sounds good, we can have this pull request merged, and I will rebase my branch based on it.